src/utils/Sort.js

```diff
@@ -640,6 +643,17 @@ export function cardGetLabels(card, sort, showOther) {
     ret = [cardDevotion(card, 'g').toString()];
   } else if (sort === 'Unsorted') {
     ret = ['All'];
+  } else if (sort === 'Popularity (Cube Inclusion Percentage)') {
+    const popularity = Math.round(card.details.popularity);
+    if (popularity < 1) ret = ['0-1%'];
+    else if (popularity < 2) ret = ['1-2%'];
+    else if (popularity < 5) ret = ['3-5%'];
+    else if (popularity < 8) ret = ['5-8%'];
+    else if (popularity < 12) ret = ['8-12%'];
+    else if (popularity < 20) ret = ['12-20%'];
+    else if (popularity < 30) ret = ['20-30%'];
+    else if (popularity < 50) ret = ['30-50%'];
+    else if (popularity < 100) ret = ['50-100%'];
```

Comment:
Just for completeness sake (and local testing) this should be `<=` for the highest bracket.